docs/api-guide/fields.md

```diff
@@ -45,6 +45,8 @@ Defaults to `True`.
 
 Normally an error will be raised if `None` is passed to a serializer field. Set this keyword argument to `True` if `None` should be considered a valid value.
 
+Note that setting this argument to `True` will imply a default value of `null` for serialization output, but does imply a default for input deserialization.
```

Comment:
Hi,

shouldn't this:
```
but does imply a default for input deserialization.
```
be instead:
```
but does not imply a default for input deserialization.
```

Sorry for the noise if I'm wrong :)

Reply:
Yep - that's definitely a typo.